client/src/components/layout/TripTile.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const TripTile = (props) => {
+  let location = ""
+  if (props.trip.city) {
```

Comment:
Just a thought, but you may consider destructuring your props here since there is a lot of `props.trip.something`. Totally not required, but may help to clean up the repetition in your code!